vmngclient/api/templates/feature_template.py

```diff
@@ -2,18 +2,21 @@
 
 from abc import ABC, abstractmethod
 from pathlib import Path
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, List, Union
 
 from jinja2 import DebugUndefined, Environment, FileSystemLoader, meta  # type: ignore
 from pydantic import BaseModel  # type: ignore
 
+from vmngclient.utils.device_model import DeviceModel
+
 if TYPE_CHECKING:
     from vmngclient.session import vManageSession
 
 
 class FeatureTemplate(BaseModel, ABC):
     name: str
     description: str
+    device_models: List[Union[DeviceModel, str]]
```

Comment:
I think that we can use only `DeviceModel` object. Pydantic should convert strings into our objects instantly.